src/main/java/Util.java

```diff
@@ -0,0 +1,61 @@
+public class Util {
```

Comment:
[Code Design] Good that you make a dedicated utility class. But I think the methods here are more suited to a separate UI class.